tardis/io/tests/test_config_reader.py

```diff
@@ -221,6 +221,46 @@ def test_plasma_nlte_section_config(tardis_config_verysimple_nlte):
     assert ve.type is ValueError
 
 
+def test_plasma_nlte_exc_section_config(tardis_config_verysimple_nlte):
+    """
+    Configuration Validation Test for Plasma Section of the Tardis Config YAML File.
+
+    Validates:
+        nlte_excitation_species: should be included in continuum_interaction
+
+    Parameter
+    ---------
+        `tardis_config_verysimple_nlte` : YAML File
+
+    Result
+    ------
+        Assertion based on validation for specified values
+    """
+    conf = Configuration.from_config_dict(
+        tardis_config_verysimple_nlte, validate=True, config_dirname="test"
+    )
+    tardis_config_verysimple_nlte["plasma"]["continuum_interaction"][
+        "species"
+    ] = [
+        "He I",
+    ]
+    tardis_config_verysimple_nlte["plasma"]["nlte_excitation_species"] = ["H I"]
+    with pytest.raises(ValueError) as ve:
+        nlte_excitation_species = tardis_config_verysimple_nlte["plasma"][
+            "nlte_excitation_species"
+        ]
+
+        for species in nlte_excitation_species:
+            if not (
+                species
+                in tardis_config_verysimple_nlte["plasma"][
+                    "continuum_interaction"
+                ]["species"]
+            ):
+                raise ValueError("Nlte excitation species not in continuum.")
+    assert ve.type is ValueError
```

Comment:
I am very confused about what this test is supposed to do. You set values for the `continuum_interaction_species` and the `nlte_excitation_species` in the config by hand. Then you check that the `nlte_excitation_species` are not in the `continuum_interaction_species`  with a piece of code that only exists in this test. This check fails because the values were set correctly in the config and you raise a `ValueError` by hand. The test, which expects a `ValueError`,  then obviously passes. As far as I can tell, the only thing that is being tested (in a very convoluted way) is that the config values exist and can be set to the desired values. I know that the test if derived from already existing tests but they also do not seem to make much more sense. Maybe I am just missing something?

Reply:
I'm not sure why this is the way to do it. I mimicked it from the existing tests. Maybe @andrewfullard would have a good answer?

Reply:
To do this properly we should follow this

def test_fails_but_bad_style():
    try:
        x = 1 / 1
        assert False
    except ZeroDivisionError:
        assert True

Reply:
I updated the tests, now they get skipped, and I'm not sure if that's a good or a bad thing. I pushed it here, so that you guys can see them too.